concepticondata/conceptlists/Maciejewski-2016-100.tsv

```diff
@@ -0,0 +1,201 @@
+Maciejewski-2016-100-120	120	pose	2	to puzzle or embarrass with a difficult problem or question			28.24	0.58	74	11.94	1.08	3.04	8	9	2.02	3.64	NA	4.56
+Maciejewski-2016-100-121	121	prop	1	to support, stabilize, or sustain with or as if with a beam, stick, stone, or the like			48.76	0.28	52	6.39	0.81	2.66	6	4	1.29	2.51	4.74	4.65
+Maciejewski-2016-100-122	122	prop	2	a piece of furniture or other article used for a theatrical presentation or the like; stage property			50.34	0.28	52	6.39	0.81	2.66	6	4	1.29	2.51	4.74	4.65
+Maciejewski-2016-100-123	123	pulse	1	the periodic throbbing of arteries that results from the beating of the heart, especially as felt in the wrist or neck	PULSE	3267	73.77	0.63	95	11.14	1.05	2.33	7	7	1.49	4.33	NA	4.86
+Maciejewski-2016-100-124	124	pulse	2	an edible seed found in the pods of legumes such as peas or lentils			26.03	0.63	95	11.14	1.05	2.33	7	7	1.49	4.33	NA	4.86
+Maciejewski-2016-100-125	125	pump	1	a mechanical or biological device for compressing a fluid or gas, or moving it from one place to another, especially through pipes or the like			69.94	0.6	82	12.59	1.1	2.26	13	11	1.54	5.03	5.56	5
+Maciejewski-2016-100-126	126	pump	2	a simple low women's shoe without buckles, laces, or the like			28.96	0.6	82	12.59	1.1	2.26	13	11	1.54	5.03	5.56	5
+Maciejewski-2016-100-127	127	pupil	1	any person who studies under a teacher	PUPIL (STUDENT)	753	56.62	0.3	31	24.72	1.39	2.04	2	3	1.28	5.72	5.7	5.47
+Maciejewski-2016-100-128	128	pupil	2	the small, dark, circular opening in the center of the eye, through which light passes to the retina	PUPIL	1658	43.38	0.3	31	24.72	1.39	2.04	2	3	1.28	5.72	5.7	5.47
+Maciejewski-2016-100-129	129	rail	1	a horizontal bar that extends between two posts and serves as a barrier, fence, or the like			82.55	0.77	82	41.17	1.61	2.96	5	15	1.43	4.99	5.4	5.3
+Maciejewski-2016-100-130	130	rail	2	to denounce someone or something in harsh or abusive terms; condemn (usually followed by against or at)			17.43	0.77	82	41.17	1.61	2.96	5	15	1.43	4.99	5.4	5.3
+Maciejewski-2016-100-131	131	rank	1	the relative position of one person or group of persons to another in a hierarchy or society			63.56	0.53	75	24.57	1.39	2.91	14	13	1.88	2.48	NA	4.71
+Maciejewski-2016-100-132	132	rank	2	having an extremely offensive odor or taste			36.44	0.53	75	24.57	1.39	2.91	14	13	1.88	2.48	NA	4.71
+Maciejewski-2016-100-133	133	ray	1	a thin beam of radiation, especially light			76.32	0.69	79	21.4	1.33	2.96	4	10	1.79	5.56	4.7	4.8
+Maciejewski-2016-100-134	134	ray	2	any of various fishes having cartilaginous skeletons, flattened bodies, large pectoral fins, and long narrow tails			22.44	0.69	79	21.4	1.33	2.96	4	10	1.79	5.56	4.7	4.8
+Maciejewski-2016-100-135	135	rear	1	the back part of something			69.8	0.55	70	29.82	1.47	3.26	9	11	1.57	3.67	4.02	4.78
+Maciejewski-2016-100-136	136	rear	2	to raise to maturity			29.97	0.55	70	29.82	1.47	3.26	9	11	1.57	3.67	4.02	4.78
```

Comment:
Maybe 2134 RAISE (A CHILD)?

Reply:
_rear_ can also refer to animals so I would not map it.

Reply:
I must admit RAISE (A CHILD) seems misleading in the way that its naming differs from its definition which also includes a possibility of raising some 'other living creature'. Maybe it should be modified so that the definition matches the name of the concept?

Reply:
If you change the definition, please check the mappings first. The crucial "definition" is: what have we linked it to so far?

Reply:
At the moment, there're 6 lists, mapped to RAISE (A CHILD) of which 4 are most likely to refer to raising children (not animals). The other two glosses are ambiguous in this sense so I guess it would make more sense to leave the definition as it is. In this case, it would fit to _rear_ in the given list. @AnnikaTjuka what do you think?

Reply:
The German mapping _aufziehen_ in Dellert-2017-1016-912 can also refer to animals. So it might be better to change the name of the concept to RAISE (BRING UP). I'll put this suggestion in an issue and map _rear_ to RAISE (A CHILD) for the time being.